NorthstarDedicatedTest/ExploitFixes.cpp

```diff
@@ -260,60 +270,29 @@ KHOOK(ReadUsercmd, ("server.dll", "4C 89 44 24 ? 53 55 56 57"), void, __fastcall
 	std::string BLOCK_PREFIX =
 		"ReadUsercmd (command_number delta: " + std::to_string(cmd->command_number - fromCmd->command_number) + "): ";
 
-	if (cmd->worldViewAngles.IsInvalid())
-	{
-		BLOCKED_INFO("CMD has invalid worldViewAngles");
-		goto INVALID_CMD;
-	}
-
-	if (cmd->attackangles.IsInvalid())
-	{
-		BLOCKED_INFO("CMD has invalid attackangles");
-		goto INVALID_CMD;
-	}
+#define FORCE_3_VALID(name)                                                                                                                \
+	if (name.IsInvalid())                                                                                                                  \
+	name = {0, 0, 0}
```

Comment:
Shouldn't `#defines` generally be at the top of the file for readability? ^^

Reply:
It's for a specific function and is not used elsewhere. This is also a .cpp file, not a header, so it cannot (should not) be imported.
I can add #undef after the function or convert it to a template function if you'd like.